src/ViewHoldingsRecord.js

```diff
@@ -329,7 +329,10 @@ class ViewHoldingsRecord extends React.Component {
     const holdingsRecord = this.getMostRecentHolding();
 
     const searchParams = new URLSearchParams(location.search);
-    searchParams.append('relatedRecordVersion', holdingsRecord._version);
+
+    if (!searchParams.has('relatedRecordVersion')) {
```

Comment:
Just as a precaution, to make sure that the parameter value is correct - let's delete the parameter and then add it. And same with `ViewInstance`